src/test/java/net/minestom/server/utils/ChunkUtilsTest.java

```diff
@@ -48,4 +57,35 @@ private static Stream<Arguments> testForDifferingChunksInRangeParams() {
     }
 
     private record ChunkCoordinate(int x, int z) {}
+
+    @Test
+    public void effectiveViewDistanceUsesServerFlagWhenNull() {
+        int result = ChunkUtils.computeEffectiveViewDistance((byte) 10, null);
```

Comment:
```suggestion
        int result = ChunkUtils.computeEffectiveViewDistance(10, null);
```
see comment on ChunkUtils.java#102